src/features/tasks/l10n/messageIds.ts

```diff
@@ -150,6 +155,14 @@ export default makeMessages('feat.tasks', {
     expired: m('Expired'),
     scheduled: m('Scheduled'),
   },
+  taskChangeCampaignDialog: {
+    dialogTitle: m('Move task'),
+    error: m('Error: Could not move the task to the selected project'),
+    moveButtonLabel: m('Move'),
```

Comment:
Note to remove this message after updating to keep it in the messages of the dialog instead of sending it in (typescript unfortunately doesn't help out by catching if a message is unused).